Source/WebKit/UIProcess/API/Cocoa/_WKWebExtension.mm

```diff
@@ -298,22 +324,30 @@ - (BOOL)_backgroundContentUsesModules
 
 #else // ENABLE(WK_WEB_EXTENSIONS)
 
-+ (instancetype)extensionWithAppExtensionBundle:(NSBundle *)appExtensionBundle
++ (void)extensionWithAppExtensionBundle:(NSBundle *)appExtensionBundle completionHandler:(void (^)(_WKWebExtension *extension, NSError *error))completionHandler
 {
-    return nil;
```

Comment:
I understand that these methods are no-ops when web extension support isn't enabled, but shouldn't we still call the completion handler here and in `+ extensionWithResourceBaseURL`?

Reply:
Yeah, I can do that. I was on the fence about it too.